File: js/stringToIntegerViceVersa.js
// Write two functions:
// toInt() : A function to convert a string to an integer
// toStr() : A function to convert an integer to a string
// https://edabit.com/challenge/EpCYjhzPKtCkkspL9

const toInt = (str) => Number(str);

const toStr = (num) => num.toString();

module.exports = {
  toInt,
  toStr
};

File: CSharp/StringToIntegerViceVersa.cs
namespace CSharp
{
    // Write two functions:
    // ToInt() : A function to convert a string to an integer
    // ToStr() : A function to convert an integer to a string
    // https://edabit.com/challenge/h7j7rTifpx7tBbzGC
    public static class StringToIntegerViceVersa
    {
        public static int ToInt(string str) => int.Parse(str);

        public static string ToStr(int num) => num.ToString();
    }
}